assets/js/blocks/handpicked-products/index.tsx

```diff
@@ -0,0 +1,42 @@
+/**
+ * External dependencies
+ */
+import { registerBlockType } from '@wordpress/blocks';
+import { getSetting } from '@woocommerce/settings';
+import { Icon, stack } from '@wordpress/icons';
+
+/**
+ * Internal dependencies
+ */
+import './editor.scss';
+import metadata from './block.json';
+import { Edit } from './edit';
+
+registerBlockType( metadata, {
+	icon: {
+		src: (
+			<Icon
+				icon={ stack }
+				className="wc-block-editor-components-block-icon"
+			/>
+		),
+	},
+	attributes: {
+		...metadata.attributes,
+		columns: {
+			type: 'number',
+			default: getSetting( 'default_columns', 3 ),
+		},
+	},
+
+	/**
+	 * Renders and manages the block.
+	 *
+	 * @param {Object} props Props to pass to block.
+	 */
```

Comment:
I know it was carried over but should we remove it? I believe the already present typescript hints provide more information and value